LeagueToolkit/LeagueToolkit.csproj

```diff
@@ -38,4 +36,11 @@
     <None Include="fantome.png" Pack="true" Visible="false" PackagePath="" />
   </ItemGroup>
 
+  <!-- allow IO.Extensions to access the internal fields/properties... -->
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
+      <_Parameter1>LeagueToolkit.IO.Extensions</_Parameter1>
+    </AssemblyAttribute>
+  </ItemGroup>
+
```

Comment:
As far as I can see, this is only necessary because the constructor for `SkeletonJoint` isn't public, but just internal. Wonder whether it's worth to just make it public instead, as 
1. there is no public constructor for `SkeletonJoint` at the moment at all, which feels weird to me
2. clearly there is a usecase for construction a `SkeletonJoint`
3. this arguable hack to access the constructor would be gone

Reply:
> This "hack" is not uncommon but normally used for tests. But I think for "extensions" it is also valid.

Generally fair point, but here I don't think there's any harm in making the constructor in point public, especially given that there is no other public constructor at all, and the constructor for `Skeleton` that takes a `List<SkeletonJoint>` is also public.

Considering that this is a library it's probably good to provide as many useful functions and constructors as possible, just excluding those which are not meant for external consumption.

Reply:
done